edi_exchange_template/views/edi_exchange_template_output_views.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="edi_exchange_template_output_view_tree" model="ir.ui.view">
+        <field name="model">edi.exchange.template.output</field>
+        <field name="arch" type="xml">
+            <tree string="EDI Exchange Template Output">
+                <field name="backend_id" />
+                <field name="name" />
+                <field name="template_id" />
+                <field name="template_key" />
+            </tree>
+        </field>
+    </record>
+    <record id="edi_exchange_template_output_view_form" model="ir.ui.view">
+        <field name="model">edi.exchange.template.output</field>
+        <field name="arch" type="xml">
+            <form string="EDI Exchange Template Output">
+                <sheet>
+                    <div class="oe_title">
+                        <label for="name" class="oe_edit_only" />
+                        <h1>
+                            <field name="name" />
+                        </h1>
+                    </div>
+                    <group name="wrapper">
+                        <group name="main">
+                            <field name="type_id" />
+                            <field name="backend_id" />
+                            <field name="template_key" />
+                            <field name="template_id" />
+                            <field name="template_arch" />
```

Comment:
If you add the template_arch outside the group it will be bigger and easier to configure